### PR TITLE
Add missing fmt examples

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
r? @QuietMisdreavus

### CODE REVIEW

File: src/libcore/fmt/mod.rs [hunk 1]
```diff
@@ -1132,6 +1132,36 @@ impl<'a> Formatter<'a> {
...
+    ///         Foo {
+    ///             nb,
+    ///         }
+    ///     }
+    /// }
+    ///
+    /// impl fmt::Display for Foo {
+    ///     fn fmt(&self, formatter: &mut fmt::Formatter) -> fmt::Result {
+    ///         // We need to remove "-" from the number output.
+    ///         let tmp = if self.nb < 0 { self.nb * -1 } else { self.nb }.to_string();
```
Comment (QuietMisdreavus) [inline src/libcore/fmt/mod.rs]: This could probably be expressed as `self.nb.abs().to_string()`.

Comment (GuillaumeGomez) [reply src/libcore/fmt/mod.rs]: This would be better actually.

**Pull request merged**